scout/server/blueprints/variant/templates/variant/cancer-variant.html

```diff
@@ -0,0 +1,671 @@
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">{{ external_links() }}</div>
+    </div>
+    <div class="row">
+      <div class="col-12">{{ activity_panel(events|reverse) }}</div>
+    </div>
+    {% if config['MAIL_USERNAME'] %}
+      {# Email setting must be setup #}
+      {{ modal_sanger() }}
+    {% endif %}
+
+    {% if config['MAIL_USERNAME'] %}
+      {# Email setting must be setup #}
+      {{ modal_cancel_sanger() }}
+    {% endif %}
+</div>
+{{ modal_causative(case, institute, variant) }}
+{% endblock %}
+
+{% macro sanger_button() %}
+  {% if variant.sanger_ordered %}
+    {% if variant.validation in ['True positive', 'False positive']%}
+      <p class="navbar-text">Sanger: {{ variant.validation }}</p>
+    {% else %}
+      <button class="btn btn-light" data-toggle="modal" data-target="#cancelSangerModal">
+        Sanger pending...
+      </button>
+    {% endif %}
+  {% else %}
+    <button class="btn btn-light" data-toggle="modal" data-target="#sangerModal">
+      Sanger
+    </button>
+  {% endif %}
+{% endmacro %}
+
+{% macro sanger_form() %}
+  <ul class="list-group">
+    <div class="list-group-item">
+      <strong>Case {{ case.display_name }}</strong><br>
+      <a href="{{ url_for('variant.variant', institute_id=institute._id,
+                          case_name=case.display_name, variant_id=variant._id) }}">
+        {{ variant.display_name }}
+      </a>
+    </div>
+    <div class="list-group-item">
+      <strong>HGNC symbols</strong>:
+      {{ variant.hgnc_symbols|join(', ') }}
+    </div>
+    <div class="list-group-item">
+      <strong>Gene panels</strong>:
+      {{ variant.panels|join(', ') }}
+    </div>
+    <div class="list-group-item disabled"><strong>Amino acid changes</strong></div>
+    {% for gene in variant.genes %}
+        {% for transcript in gene.transcripts %}
+          {% if "refseq_identifiers" in transcript or transcript.is_canonical %}
+              {% if transcript.protein_sequence_name%}
+                <div class="list-group-item">
+                  [{{ transcript.refseq_identifiers|join(", ") }}]<span class="text-muted"> ({{ transcript.transcript_id }})</span>:
+                  {{ (transcript.protein_sequence_name or '')|url_decode }}
+                </div>
+              {% endif %}
+            {% endif %}
+          {% endfor %}
+    {% endfor %}
+    <div class="list-group-item disabled"><strong>GT call</strong></div>
+    {% for individual in variant.samples %}
+      <div class="list-group-item">
+        {{ individual.display_name }}: {{ individual.genotype_call }}
+      </div>
+    {% endfor %}
+    <div class="list-group-item">
+      <strong>Ordered by</strong>:
+      {{ current_user.name }}
+    </div>
+    <div class="list-group-item">Comment: <input type="text" size=45 name="verification_comment"></div>
+  </ul>
+{% endmacro %}
+
+{% macro matching_variants() %}
```

Comment:
This is exactly the same code as in variant. Instead of repeat it it would be much better to move it somewhere else and import it, same in variant template. So it can be shared by the 2 types of variants